### ISSUE TITLE
Cloudflare SSL and Wordpress wp-admin access (ERR_TOO_MANY_REDIRECTS redirect loop)

### ISSUE THREAD

**Issue by jonathannz** [engintron/engintron#758]
Hi,
In my usage scenario, the WHM/Cpanel server has a decidate IP addresse. We only have two sites on there (our own), and they use this IP addresse.

I followed this instruction:

> If you use SSL via CloudFlare, make sure you choose "Flexible SSL" inside "Crypto" settings. This way all HTTPS traffic from CloudFlare will be proxied to your Nginx HTTP port and thus you'll benefit both in terms of caching and overall performance.

But this now results in wp-admin being inaccessible, due to `ERR_TOO_MANY_REDIRECTS` error.

Can someone suggest what to do about this?

Thanks.

**Comment by jonathannz** [comment]
Actually, it looks like the issue is bigger than I original thought. I guess Cloudflare, etc., was taking some time to take effect.

If I set the Crypto SSL setting to `Flexible` it throws the sites into a redirect loop (I gather it's an HTTPS > HTTP loop). I have no idea where to resolve that. Any suggestions?

As far as I can tell, the only way I get the sites to load correctly is if I set the Crypto SSL to `Full (Strict)`. But I gather that then circumvents the benefits of having Nginx on the server, right?

It've looked in the .htaccess files of the two sites we run, and there's nothing there doing rewrites for HTTP / HTTPS. So I am guessing it's occurring on the Apache server config level, in WHM? Not sure.

**Comment by tux-tn** [comment]
Hello @jonathannz,
Are you setting the $PROXY_DOMAIN_OR_IP value to your dedicated IP address in Engintron custom_rules file?

**Comment by fevangelou** [comment]
It's all here: https://github.com/engintron/engintron/wiki/Engintron-&-CloudFlare

**Comment by jonathannz** [comment]
@tux-tn Yes, I've set the dedicated ip in the custom_rules. Thanks.

@fevangelou Thanks Fotis. I've had a look at this link and see if it suggests anything more than what's I've already attempted.

**Comment by jonathannz** [comment]
Okay, so that's the page I already read through as part of implementing Engintron. It's what I quote from in my OP. I am I didn't give a direct link to make that clearer.

In my scenario, there is one IP, and it has two domains (sites) on it.

I added the IP to the `$PROXY_DOMAIN_OR_IP` setting in the custom_rules.

If I set CF to Flexible SSL, the site ends up in a redirect loop.

I'll try it all again, and see what happens.
